js/events/utils/event_nodes_disposing.js

```diff
@@ -0,0 +1,20 @@
+import eventsEngine from '../core/events_engine';
+
+const REMOVE_EVENT_NAME = 'dxremove';
```

Comment:
Import this event name from the [initial file](https://github.com/DevExpress/DevExtreme/blob/20_2/js/core/remove_event.js) be better in this case.